.styleci.yml

```diff
@@ -0,0 +1,18 @@
+preset: symfony
```

Comment:
Should this file be excluded in .gitattributes?

Reply:
How about removing the entire file, and instead of using StyleCI, providing an easy means to run coding style fixes as a developer? There's also the problem of having two different configurations now - one for Travis CI, one for StyleCI.

Reply:
There would be a nice solution (like implemented by php-http): using a bridge component that allows style-ci to be defined, and php-cs-fixer to be able to read it.
It is called php-cs-fixer-bridge or something similar.

But still. we could keep the two files, as a CS change is quite improbable after we agreed on a standard. WDYT?

Reply:
I think we should start with this new file and open a follow-up issue for further discussion.
I don't want to block 4.0.0-alpha.1

Reply:
Ok! great